helm-chart/renku/values.yaml

```diff
@@ -374,6 +374,7 @@ redis:
       appendonly no
       # Disable RDB persistence.
       save ""
+  
```

Comment:
not that is super important, but was this new line intended?

Reply:
no...

Reply:
haha now I'm confused... I removed it but somehow it's still there 🤷

Reply:
there's also 2 white spaces?...

Reply:
🤦